src/mixins/common-mixin.ts

```diff
@@ -119,9 +119,9 @@ export default class CommonMixin extends Vue {
         return 'Record Conversion'
       case RouteNames.CORRECTION:
         if (this.isCorrectedIncorporationApplication) {
-          return 'Correction - Incorporation Application'
+          return 'Register Correction - Incorporation Application'
```

Comment:
I'm not certain what the requirement is, but I think it doesn't include the IA correction title. Please confirm with Linda/Dave and Yui. Thanks.

Reply:
Okay I'll ask during stand up to confirm

Reply:
I heard in grooming today that it's "Register Correction" for all correction types, so this code can be simplified.